metacat-main/src/main/java/com/netflix/metacat/main/services/search/ElasticSearchDoc.java

```diff
@@ -51,11 +53,13 @@
 
     private String id;
     private Object dto;
+    private final String timestamp = String.valueOf(Instant.now().toEpochMilli());
```

Comment:
Can we set the timestamp in the constructor and have this as null by default

Reply:
Why do we want to put null as default? This field is always required.

Reply:
When we read the document from elasticsearch, how will you populate this Object?

Reply:
I am not aware of this use-case.Needs a bit explain how to use it after reading from the es. If we need to update a certain field, we can upload the fields explicitly.

Reply:
Agreed. Moved timestamp init to constructor.